core/rawdb/schema.go

```diff
@@ -92,7 +92,7 @@ var (
 	blockBodyPrefix     = []byte("b") // blockBodyPrefix + num (uint64 big endian) + hash -> block body
 	blockReceiptsPrefix = []byte("r") // blockReceiptsPrefix + num (uint64 big endian) + hash -> block receipts
 
-	blobSidecarsPrefix = []byte("s") // blobSidecarsPrefix + num (uint64 big endian) + hash -> sidecars
+	BlobSidecarsPrefix = []byte("s") // blobSidecarsPrefix + num (uint64 big endian) + hash -> sidecars
```

Comment:
Why do we need to export this variable?

Reply:
At first I decided to use it to iterate back the blobs in db, but I forgot to redo it. Fixed.